src/config/2019.json

```
@@ -829,6 +829,14 @@
       "github": "noahblon",
       "twitter": "noahblon"
     },
+    "noah-vdv": {
+      "name": "Noah van der Veer",
+      "teams": [
+        "translators"
+      ],
+      "github": "noah-vdv",
```

If you want your GitHub profile picture then can add it here:

```suggestion
      "avatar_url": "https://avatars1.githubusercontent.com/u/46689362?v=4&s=200",
      "github": "noah-vdv",
```

Otherwise you get a random one of our characters as shown on the test site: https://20201215t151713-dot-webalmanac.uk.r.appspot.com/nl/2020/contributors?teams=translators#noah-vdv